.github/workflows/mesh.b2b.deployed.yml

```diff
@@ -0,0 +1,37 @@
+name: B2B Web SDK - Test & Deploy
+on:
+  release:
+    types: [published]
+jobs:
+  build-and-test:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout repo
+        uses: actions/checkout@v3
+      - name: Setup Node.js environment
+        uses: actions/setup-node@v3
+        with:
+          node-version: 18.x
+      - name: Install dependencies
+        run: yarn
+      - name: Checking for type errors
+        run: yarn typecheck
+      - name: Checking for linting errors
+        run: yarn lint
+      - name: Yarn test
+        run: yarn test
+  deploy:
+    runs-on: ubuntu-latest
+    needs:
+      - build-and-test
+    steps:
+      - uses: actions/checkout@v4
+      # Setup .npmrc file to publish to npm
+      - uses: actions/setup-node@v3
+        with:
+          node-version: '18.x'
+          registry-url: 'https://registry.npmjs.org'
+      - run: yarn
+      - run: yarn publish:npm
```

Comment:
Is yarn publish:npm working from the root folder? A far as I remember it had to be run from the package folder in earlier version.

Reply:
This is being taken care by lerna, which simultaneously triggers the queue from `package` folder, so every subfolder in package will get this command triggered